services/metadata/src/metadata/routes/domain/template.clj

```diff
@@ -105,3 +105,8 @@
 
    :name
    (describe String "The metadata template name")})
+
+(s/defschema CSVDownloadQueryParams
+  (assoc StandardUserQueryParams
```

Comment:
This doesn't really require the user, so I wouldn't be opposed to removing it. But, it's nice for logs to know who requested something, so I kept it in for that reason.